app/helpers/selections_helper.rb

```diff
@@ -1,2 +1,50 @@
+require 'json'
+require 'ostruct'
+
 module SelectionsHelper
+
+  def get_forecast(stations)
+
+    w_api = Wunderground.new("dda1e42fba07eb1c")
+    current_user.forecast = [];
+
+    @forecasts = []
```

Comment:
The instance variable should be created in the controller. This method should return the array. The controller should set `@forcasts` with this value.